closure/testing/test/BUILD

```diff
@@ -159,3 +199,4 @@ closure_js_test(
 #     name = "darwin",
 #     values = {"cpu": "darwin"},
 # )
+
```

Comment:
Do we expect that all original tests work with rules_webtesting ? If so we should maybe introduce a short macro that create two tests one to execute the test on phantomJs and another to execute it on headless chrome . WDT ?

Reply:
the original tests work with rules_webtesting. I can introduce something like closure_js_test_suite. But if we are deprecating phantomJS, then I think the combined macro shouldn't be encouraged

Reply:
I was thinking about an internal macro just for closure/testing/test

Reply:
Let's not do that in the scope of this PR. We can reconsider that in the future.